docs/sources/explore/trace-integration.md

```diff
@@ -18,7 +18,7 @@ Supported data sources are:
 
 For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.
 
-{{< figure src="/static/img/docs/v70/explore-trace-view-full.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
+{{< figure src="/static/img/docs/v80/explore-trace-view-full.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
```

Comment:
Please put the image in the explore folder in the website repo, not v80.

Reply:
This applies to all images. Put the images in the subject folder, not a version folder.

Reply:
I assume the old ones that did not change here can stay as they are or should they also be changed?